README.md

```diff
@@ -1,7 +1,7 @@
 # Overview
-This repository provides various Protocol Buffer extensions for the Go
-language (golang), namely support for record length-delimited message
-streaming.
+This repository provides various Protocol Buffer feature extensions for the Go
+programming language (golang), namely support for record length-delimited 
+message streaming.
```

Comment:
Oh, with the next protobuf release, the `protodelim` package should provide the exact same functionality, AFAICT? See https://github.com/protocolbuffers/protobuf-go/commit/fb0abd915897428ccfdd6b03b48ad8219751ee54

It would be interesting to compare the two implementations in the short term (to fix issues, if any), and possibly your repository could be retired in favor of the official protodelim package in the long term?

Reply:
I'll give `protodelim` a deeper look in the future.  I'm a bit surprised considering that the bug I originally filed for this about 11 years ago now was never pinged.  There are a few things about how the modern Protocol Buffer API works (in terms of support package API) I will need to develop a deeper understanding of.

Reply:
The team has changed enough that I strongly suspect nobody in the current team has had any context on your interaction from 11 years ago :)